src/observable/fromEvent.ts

```diff
@@ -1,40 +1,60 @@
 import {Observable} from '../Observable';
+import {isEventTarget} from '../util/isEventTarget';
+import {isHTMLCollection} from '../util/isHTMLCollection';
+import {isNodeList} from '../util/isNodeList';
 import {tryCatch} from '../util/tryCatch';
 import {errorObject} from '../util/errorObject';
 import {Subscription} from '../Subscription';
 import {Subscriber} from '../Subscriber';
 
+export type EventEmmitterLike1 = {
```

Comment:
I like those names, would like another suggestion since I'm not great for naming in English in general ;) @blesh @staltz , would it be ok with above suggested names?